CHANGES.md

```diff
@@ -72,6 +72,8 @@ To be released.
  -  Marked `Address` and `HashDigest` as readonly.  [[#610]]
  -  `IceServer.CreateTurnClient()` became to throw `ArgumentException` when
     received invalid url.  [[#622]]
+ -  `Swarm<T>.ProcessMessageAsync()` became to throw `InvalidMessageException`
+    when received incomprehensible message.  [[#628], [#641]]
```

Comment:
Since `Swarm<T>.ProcessMessageAsync()` is a private method, it doesn't need to be written to the changelog.

Reply:
Then, is the changelog needed? 🤔 
I'm not sure why(how) we should mention #628 on bug-fix of private method.

Reply:
I think this doesn't need to be written to the changelog.

Reply:
How about this?
```
Fixed bug that message processing is halted when received an invalid type of message.
```

Reply:
Awesome 😎 . But the issue looks like described it can make whole processes be halted.
I will write changelog again following your format, thanks.